components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/ConfigurationManagerImpl.java

```diff
@@ -263,13 +266,26 @@ public Resource replaceResource(String resourceTypeName, ResourceAdd resourceAdd
         return resource;
     }
 
+    @Override
+    public Resource replaceResource(String resourceTypeName, Resource resource)
+            throws ConfigurationManagementException {
+
+        validateResourceReplaceRequest(resourceTypeName, resource);
+        String resourceId = generateResourceId(resourceTypeName, resource.getResourceName());
+        resource.setResourceId(resourceId);
+        this.getConfigurationDAO().replaceResourceWithFiles(resource);
+        if (log.isDebugEnabled()) {
+            log.debug(resource.getResourceName() + " resource created successfully.");
+        }
+        log.info(resource.getResourceName() + " resource created successfully.");
```

Comment:
Info log is enough, no need both info and debug logs.

Reply:
Fixed in d8a058a